common-crafting.lic

```diff
@@ -181,4 +181,38 @@ module DRCC
       DRC.bput("stow my #{logbook} logbook", 'You put', 'What were you referring to')
     end
   end
+
+  def order_enchant(stock_room, stock_needed, stock_number)
+    stock_needed.times do
+      order_item(stock_room, stock_number)
+      stow_crafting_item(left_hand, @bag, @belt)
+      stow_crafting_item(right_hand, @bag, @belt)
+      next unless left_hand && right_hand
+    end
+  end
+
+  def fount(stock_room, stock_needed, stock_number, quantity)
```

Comment:
💡 Could add a verb to this function name? What is it doing with a fount?

Reply:
This function checks for the fount, either on a belt, in a container, or on a brazier. Then, the fount count is found and will discard if below a certain threshold and purchase a new one.